fr/intro/conventions.rst

```diff
@@ -90,34 +90,29 @@ Conventions pour les Models et les Bases de Données
 Les noms de classe de model sont au pluriel et en CamelCase. People, BigPeople,
 et ReallyBigPeople en sont des exemples.
 
-Les noms de tables correspondant aux models CakePHP sont au pluriel et
-utilisent le caractère souligné (underscore). Les tables correspondantes
-aux models mentionnés ci-dessus seront donc respectivement : ``people``,
-``big_people`` et ``really_big_people``.
+Les noms de tables correspondant aux models CakePHP sont au pluriel et utilisent
+le caractère souligné (underscore). Les tables correspondantes aux models
+mentionnés ci-dessus seront donc respectivement : ``people``, ``big_people`` et
+``really_big_people``.
 
-Note des traducteurs francophones : seul le dernier mot est au pluriel et
-tous les pluriels français ne seront pas compris par CakePHP sans lui indiquer
-précisément (par exemple cheval/chevaux). Voir pour cela le chapitre sur les
-inflexions.
-
-Pour vous assurer de la syntaxe des mots pluriels et singuliers, vous pouvez
-utiliser la bibliothèque utilitaire :php:class:`Inflector`. Voir la
-documentation sur :doc:`/core-libraries/inflector` pour plus
-d'informations.
+Note des traducteurs francophones : seul le dernier mot est au pluriel et tous
```

Comment:
You're right, maybe we can write a more generic thing for all languages that we could also add to the english version, ie:

"The convention is to use english words for table and column names. If you use words in your language, CakePHP might not be able to process the right inflections (from singular to plural and vice-versa).
For some reason, you might need to add your own language rules for some words and for that, you can use the php:class:`Inflector` utility class. Besides defining those custom inflection rules, this class will also allows you to check that CakePHP understand your custom syntax for plurals and singulars words. See the documentation about :doc:`/core-libraries/inflector` for more information."

Let me know if this needs modifications